README.md

```diff
@@ -141,16 +141,41 @@ A number of changing nodes can be simulated with the following options. The node
 The options `--sph` and `--sp` show and dump an OPC Publisher configuration file (default name: `pn.json`) that matches the configuration. In addition, a web server hosts the file on a configurable port (`--wp`, default 8080): e.g. http://localhost:8080/pn.json
 Additionally, you can set the configuration file name via the option `--spf`.
 
-## Complex type (boiler)
+## Complex type (Boiler #1)
  
 Adds a simple boiler to the address space.
 
 Features:
-- BoilerStatus is a complex type that shows: temperature, pressure and heater state
+- BoilerStatus is a complex type that shows: Temperature, pressure and heater state
 - Method to turn heater on/off
 - When the heater is on, the bottom temperature increases by 1 degree/s, the top temperature is always 5 degrees less than the bottom one
 - Pressure is calculated as 100000 + bottom temperature
 
+## Boiler #2 derived from the Device Information (DI) companion spec
+ 
+Adds a configurable boiler that exposes DI properties such as AssetId (ITagNameplate, IVendorNameplate) and DeviceHealth.
+
+Features:
+- Configure and expose: Base temperature, target temperature, temperature change speed, current temperature, heater state, overheated state, miantenance
+- Method to switch heater on/off
+- The maintenance and overheated intervals both trigger events
+
+Simulation details:
+- When the heater is off, the temperature falls to the base temperature and the heater is switched on
+- When the heater is on, the temperature raises to the target temperature and the heater is switched off
+- The overheat interval increases the temperature to 10 degrees above the overheated temperature threshold, the heater is switched off and:
+  - Emits a "CheckFunctionAlarmType" event when DeviceHealth is CHECK_FUNCTION
+  - Emits a "FailureAlarmType" event when DeviceHealth is FAILURE
+  - Emits an "OffSpecAlarmType" event when DeviceHealth is OFF_SPEC
+  - Emits a "MaintenanceRequiredAlarmType" when DeviceHealth is MAINTENANCE_REQUIRED
+
+DeviceHealth (DeviceHealthEnumeration) details:
+- NORMAL: Base temperature < temperature < target temperature

```

Comment:
>NORMAL: Base temperature < temperature < target temperature [](http://example.com/codeflow?start=2&length=59)

add equals:
Base temperature <= temperature <= target temperature to be consistent with code

Reply:
Txs